packages/airnode-validator/test/validatorTests/minimal.json

```diff
@@ -23,7 +23,8 @@
     "nodeVersion": "0.3.0",
     "cloudProvider": {
       "type": "aws",
-      "region": "us-east-1"
+      "region": "us-east-1",
+      "name": "aws-us"
```

Comment:
aws cloud provider doesn't have a name https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/types.ts#L376

Reply:
Has any of the providers `name`?